.bazelrc

```diff
@@ -65,13 +70,5 @@ build:windows --host_copt=-DWIN32_LEAN_AND_MEAN
 build:windows --copt=-DNOGDI
 build:windows --host_copt=-DNOGDI
 
-# Google cache read-only access.
-build --remote_cache=https://storage.googleapis.com/reboot-dev-bazel-remote-cache-eventuals-us
-build --remote_upload_local_results=false
-# Default timeout of 60 seconds is too long and bazel doesn't try to build
-# things locally while waiting for the timeout.
-# 10s was chosen arbitrarily as a not-too-long time by Folming and CodingCanuck.
-build --remote_timeout=10s
```

Comment:
Remind me again why we need to turn the remote cache off?

(This sort of thing would be good to document in the PR description in the future)